ruby_programming/basic_ruby/project_building_blocks.md

```diff
@@ -1170,4 +1173,4 @@ Next, make sure your method can handle multiple words:
 * [Ifeanyichukwu Amajuoyi's solution](https://github.com/Anyitechs/The_odin_project)
 * [Ashley Cook's Solution](https://github.com/Cooky90/SubStrings/blob/master/Substrings.rb)
 
```

Comment:
[Ernest Apondo's solution](https://github.com/ApondoErnest/Rock-Paper-Scissors)

Reply:
@ApondoErnest This isn't the correct method for adding your solutions. Please watch this video and follow the instructions there for creating a new pull request to add your solution: https://www.youtube.com/watch?v=V74l_zS1x8E